hack/common.sh

```diff
@@ -23,7 +23,7 @@ ARCHITECTURE=$(uname -m)
 
 function build_func_tests() {
     mkdir -p "${TESTS_OUT_DIR}/"
-    GOPROXY=off GOFLAGS=-mod=vendor \
+    GOPROXY=off GOFLAGS="$GOFLAGS -mod=vendor" \
```

Comment:
Since we export above `GOFLAGS`, would this line be expanded to something like `$additionalargs -mod=vendor -mod=vendor`?

Reply:
Right, good point. Fixed that.

Reply:
Right, good point. Fixed that.

Reply:
:+1: 

@vasiliy-ul since suse seems to use this build flow which we don't use anymore in our CI, would you be interested in writing a CI prow-job which would do basic sanity checks on the pure-go build scripts?

Reply:
@rmohr :+1: Yes, that might be useful. I am definitely interested. I will only need to get more familiar with how kubevirt CI is setup here. Could you perhaps point me where to look at regarding that topic?

Reply:
It would right now mostly mean creating a prow job like this: https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L373. 

Should not be too hard. What I would love to see is that people which rely on that are active on the project and can help/follow up on PRs with issues around it. That would be the pre-requisite for having such jobs executed.

What do you think?

Reply:
I agree. It makes sense to introduce the job only if there is someone who *actively* maintains it. No objections here.